devicetypes/timevalve-gaslock-t-08/timevalve-smart.src/timevalve-smart.groovy

```diff
@@ -0,0 +1,281 @@
+metadata {
+        definition (name: "Timevalve Smart", namespace: "timevalve.gaslock.t-08", author: "ruinnel") {
+                capability "Valve"
+                capability "Refresh"
+                capability "Battery"
+                capability "Notification"
+				capability "Configuration"
+                capability "Temperature Measurement"
+                
```

Comment:
"Temperature Measurement" are using for the notification function. Temperature value is not important.
and.. I can't find proper capability.